execution/withdrawals.md

```diff
@@ -27,13 +27,13 @@ The following sections of [ethereum/eip-4895](https://eips.ethereum.org/EIPS/eip
 
 The `withdrawals` in an execution payload are processed **after** any user-level transactions are applied.
 
-For each `withdrawal` in the list of `execution_payload.withdrawals`, the implementation must construct and execute an EVM transaction as follows:
+The implementation must construct and execute one system EVM transaction as follows:
 
 ```
 sender: SYSTEM_SENDER
-max_priority_fee_per_gas: TBD
-max_fee_per_gas: TBD
-gas_limit: TBD
+max_priority_fee_per_gas: 0
+max_fee_per_gas: 0
+gas_limit: 1600000
```

Comment:
Might be too low for complex logic

Reply:
Right. ERC20 transfers can require 50-200k gas. With 16 widrawals per block * 200k = 3.2M. Will increase to 30,000,000 (= block gas limit)  as an arbitrary higher value.